projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/torch_ods_gen.py

```diff
@@ -720,6 +720,8 @@ def emit_with_mutating_variants(key, **kwargs):
     emit("aten::bmm : (Tensor, Tensor) -> (Tensor)")
     emit("aten::cumsum : (Tensor, int, int?) -> (Tensor)")
     emit("aten::cumprod : (Tensor, int, int?) -> (Tensor)")
+    emit("aten::logcumsumexp : (Tensor, int) -> (Tensor)")
+    emit("aten::_logcumsumexp : (Tensor, int) -> (Tensor)")
```

Comment:
You don't need to add the trailing underscore variant of the op manually. It will get generated automatically. Please remove this.